homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,31 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    
+    int fibN = 0;
+    if(sequence > 1) {
+        int fibMinus2 = 0;
+        int fibMinus1 = 1;
+        for(int i = 2; i <= sequence; ++i) {
+            fibN = fibMinus1 + fibMinus2;
+            fibMinus2 = fibMinus1;
+            fibMinus1 = fibN;
+        }
+    }    
+    else {
+        return sequence;
+    }
```

Comment:
else redundant.  Move this to the top and write like:

`if (sequence < 2) {
    return sequence;
}`